projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectInformationService.java

```diff
@@ -74,259 +58,57 @@ public List<ProjectPreview> queryPreview(String filter, int offset, int limit,
-      Objects.requireNonNull(specimen);
-    }
-
-    ProjectId id = ProjectId.parse(projectId);
-    Optional<Project> optionalProject = projectRepository.find(id);
-
-    optionalProject.ifPresentOrElse(
-        project -> loadActiveExperimentForProject(project)
-            .ifPresentOrElse(addSpecimenToExperimentAndSave(specimens),
-                () -> addExperimentToProjectService
-                    .addExperimentToProject(project.getId(),
-                        "Experiment 0",
-                        List.of(),
-                        List.of(),
-                        List.of(specimens))),
-        () -> {
-          throw new ProjectManagementException("There is no project with id " + id.value());
-        }
-    );
-  }
-
-  private Consumer<Experiment> addSpecimenToExperimentAndSave(Specimen[] specimens) {
-    return activeExperiment -> {
-      activeExperiment.addSpecimens(List.of(specimens));
-      experimentRepository.update(activeExperiment);
-    };
-  }
-
-  /**
-   * Adds analytes to the active experiment of a project. If no experiment is active, a new
-   * experiment is created and set as the active experiment.
-   *
-   * @param projectId the project for which to add the species
-   * @param analytes  the analytes to add
-   * @see Experiment#addAnalytes(Collection)
-   */
-  public void addAnalyteToActiveExperiment(String projectId, Analyte... analytes) {
-    if (analytes.length < 1) {
-      return;
-    }
-    Arrays.stream(analytes).forEach(Objects::requireNonNull);
-
-    ProjectId id = ProjectId.parse(projectId);
-    Optional<Project> optionalProject = projectRepository.find(id);
-
-    optionalProject.ifPresentOrElse(
-        project -> loadActiveExperimentForProject(project)
-            .ifPresentOrElse(addAnalytesToExperimentAndSave(analytes),
-                () -> addExperimentToProjectService.addExperimentToProject(project.getId(),
-                    "Experiment 0",
-                    List.of(analytes),
-                    List.of(),
-                    List.of())),
-        () -> {
-          throw new ProjectManagementException("There is no project with id " + id.value());
-        }
-    );
-  }
-
-  private Consumer<Experiment> addAnalytesToExperimentAndSave(Analyte[] analytes) {
-    return activeExperiment -> {
-      activeExperiment.addAnalytes(List.of(analytes));
-      experimentRepository.update(activeExperiment);
-    };
-  }
-
   @PostAuthorize("hasPermission(returnObject,'VIEW_PROJECT')")
-  public Optional<Project> find(ProjectId projectId) {
-    log.debug("Search for project with id: " + projectId.toString());
-    return projectRepository.find(projectId);
+  public Project loadProject(ProjectId projectId) {
+    return projectRepository.find(projectId).orElseThrow(() -> new ProjectManagementException(
+            "Project with id" + projectId.toString() + "does not exit anymore")
+        // should never happen; indicates dirty removal of project from db
+    );
```

Comment:
As discussed in person, this should stay private and the view components should have access to the public method `Optional<Project> find(ProjectId projected)`